pkg/scheduler/util/utils_test.go

```diff
@@ -252,3 +297,47 @@ func TestPatchPodStatus(t *testing.T) {
 		})
 	}
 }
+
+func TestIsScalarResourceName(t *testing.T) {
+	tests := []struct {
+		name         string
+		resourceName v1.ResourceName
+		want         bool
+	}{
+		{
+			name:         "Should be true with extended resources name",
+			resourceName: "example.com/foo",
+			want:         true,
+		},
+		{
+			name:         "Should be false with not extended resources name",
+			resourceName: "requests.foo",
+			want:         false,
+		},
+		{
+			name:         "Should be true with hugePage resource name",
+			resourceName: "hugepages-2Mi",
+			want:         true,
+		},
+		{
+			name:         "Should be false with not HugePage resource name",
```

Comment:
Feel the name and case seems not related？

HugePage resource name is one of the filter. right?

Reply:
Thank you for your comments.
I renamed it and added some new test cases for the non-scalar-resources name.